ConnectorNavisworks/ConnectorNavisworks/Bindings/ConnectorNavisworksBindings_Events.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
```

Comment:
Finessing, but to align to other connectors we should maybe make the underscore into a `.`